src/met-atlas-viewer.js

```diff
@@ -39,130 +40,102 @@ import { makeIndexSprite } from './helpers';
  *     viewer should be placed.
  * @returns {Object} A control object with functions for controlling the viewer.
  */
-function MetAtlasViewer(targetElement) {
-  const container = document.getElementById(targetElement)
+const MetAtlasViewer = (targetElement) => {
+  const container = document.getElementById(targetElement);
 
   // Camera variables
   let fieldOfView = 90;
   let aspect = container.offsetWidth / container.offsetHeight;
   let near = 1;
   let far = 10000;
-  var camera = new PerspectiveCamera(fieldOfView, aspect, near, far)
+  let camera = new PerspectiveCamera(fieldOfView, aspect, near, far);
   camera.position.z = 3000;
   let nodeSelectCallback, updateCameraCallback;
 
-  var cameraDefault = {
+  const cameraDefault = {
     position: Object.assign({}, camera.position),
     up: Object.assign({}, camera.up),
-    target: Object.assign({}, camera.up)
+    target: Object.assign({}, camera.up),
   };
-  var flyTarget = {
+  const flyTarget = {
     active: false,
     start: undefined,
     startUp: undefined,
     end: undefined,
     target: undefined,
     startTime: undefined,
     targetUp: undefined,
-    runTime: 750
+    runTime: 750,
   };
 
-  // Set default colors
-  var nodeDefaultColor = [255,255,255];
-  var connectionStartColor = [0, 127, 255];
-  var connectionEndColor = [0, 127, 0];
-  var nodeSelectColor = [255, 0, 0];
-  var connectionSelectColor = [255, 255, 0];
-  var hoverSelectColor = [255, 0, 255];
-  var hoverConnectionColor = [255, 0, 0];
-
   // Create the scene and set background
-  var scene = new Scene();
-  scene.background = new Color( 0xdddddd );
+  let scene = new Scene();
+  scene.background = new Color(0xdddddd);
```

Comment:
Should this value be in helpers.js as well?

Reply:
Good idea. Addressed in: https://github.com/MetabolicAtlas/3d-network-viewer/pull/51/commits/f7661ce692bb7b6b94229e36185a021bd2516ac5